test/relations.test.js

```diff
@@ -1704,8 +1794,16 @@ describe('relations', function() {
         should.not.exists(err);
         c.jobs(function(err, jobs) {
           should.not.exists(err);
-          jobs.should.have.length(2);
-          done();
+          if (isCassandraConnector) {
+            var types = ['widget', 'tool', 'book'];
+            jobs.should.have.length(3);
```

Comment:
Why is Cassandra returning a different number of scoped items than other connectors? I find that very suspicious! Does it mean the scopes actually don't work with Cassandra?

It will be probably best to skip this entire test for Cassandra.